docs/src/references/mqtt-api.md

```diff
@@ -231,24 +231,25 @@ graph LR
 
-|Segment|Description|
-|-------|-----------|
-|&lt;category&gt;|Category of data (telemetry or commands) which is related to the **identifier**. A fixed list of categories is used to represent data types such as; measurements, alarms, events, commands etc.|
-|&lt;type&gt;|A unique type/name used to identify the information being pushed or received. Types allow users to filter/subscribe to data which interests them. For example, a measurement could be published under a type called "flow_rate", then other clients interested in the flow rate can subscribe to that single typed topic.|
-|`...`|Additional channel specific topic segments. Each **category** is responsible for defining the number and meaning of the remaining topic segments.|
+| Segment          | Description                                                                                                                                                                                                                                                                                                               |
+|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| &lt;category&gt; | Category of data (telemetry or commands) which is related to the **identifier**. A fixed list of categories is used to represent data types such as; measurements, alarms, events, commands etc.                                                                                                                          |
+| &lt;type&gt;     | A unique type/name used to identify the information being pushed or received. Types allow users to filter/subscribe to data which interests them. For example, a measurement could be published under a type called "flow_rate", then other clients interested in the flow rate can subscribe to that single typed topic. |
+| `...`            | Additional channel specific topic segments. Each **category** is responsible for defining the number and meaning of the remaining topic segments.                                                                                                                                                                         |
 
 
 #### Categories
 
 The following is an overview of the channel categories which are available.
 
-|Category|Description|
-|--------|-------|
-|m|Measurements|
-|e|Events|
-|a|Alarms|
-|cmd|Commands|
-|twin|Entity twin metadata|
+| Category | Description          |
+|----------|----------------------|
+| m        | Measurements         |
+| e        | Events               |
+| a        | Alarms               |
+| cmd      | Commands             |
+| twin     | Entity twin metadata |
+| status   | Service status       |
```

Comment:
AFAIK currently we have only services publishing health status messages, and this `status` category is not clearly defined:

- can health status messages be published for devices as well?
- are there any other statuses valid for devices but not services?

I think this `status` category is only referenced here, so I'd rather be conservative with the wording here and only describe what is currently implemented, instead of trying to cover how it might be implemented in the future.